autoscheduler/frontend/src/tests/ui/App.test.tsx

```diff
@@ -23,3 +24,23 @@ test('renders without errors', async () => {
   // assert
   expect(container).toBeTruthy();
 });
+
+test('shows an error page', () => {
```

Comment:
Should add a describe block here for the condition that it should show an error page (when an invalid URL is provided)

Reply:
good point